src/settings/RequestPolicy/RequestPolicyForm.js

```diff
@@ -131,23 +132,26 @@ class RequestPolicyForm extends React.Component {
       },
       location,
     } = this.props;
-    const allowedServicePoints = requestPolicies.records
-      .find(({ id }) => location.pathname.includes(id))?.allowedServicePoints;
 
-    if (allowedServicePoints) {
-      const selectedRequestTypes = Object.keys(allowedServicePoints);
-      const rules = {};
+    if (!isAddLayer(location.search)) {
```

Comment:
A code comment would be helpful here to explain why "add" is the special case and including the service-pointy rules is the default. Otherwise, I think the condition should be `if (isEditLayer())`. IOW, putting "default" behavior inside a conditional is atypical, so it needs an explanation.